src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -79,20 +99,24 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
 .pf-c-description-list__term,
 .pf-c-description-list__description {
   min-width: 0; // this allows overflow-wrap to work
+  line-height: var(-pf-c-description-list__description--LineHeight);
   text-align: left;
   overflow-wrap: break-word;
 }
 
 .pf-c-description-list__term {
   font-size: var(--pf-c-description-list__term--FontSize);
   font-weight: var(--pf-c-description-list__term--FontWeight);
+  line-height: var(--pf-c-description-list__term--LineHeight);
 
   .pf-c-description-list__text {
     display: inline;
   }
 }
 
 .pf-c-description-list__text {
+  line-height: var(--pf-c-description-list__text--LineHeight);
```

Comment:
You can just use `line-height: inherit` here, or `line-height: var(--pf-c-description-list__text--LineHeight, inherit)` and remove `--pf-c-description-list__text--LineHeight` above, if it's needed.